scripts/airtable-auto.py

```diff
@@ -0,0 +1,72 @@
+#! /usr/bin/env python3
+
+from airtable import Airtable
+import json
+import os
+
+# Write all values in the airtable table specified by
+# airtable_base_key, view_name, and table_name
+# to the file at `path`
+# needed_fields specifies the column names that each row
+# of the table must have an entry in
+
+def update_data_file(airtable_base_key, view_name, table_name, needed_fields, path):
+    try:
+        airtable = Airtable(
+            airtable_base_key,
+            table_name,
+            api_key=os.environ["AIRTABLE_API_KEY"],
+        )
+    except KeyError:
+        print("Couldn't find airtable api key environment variable")
+        exit(1)
+
+    value_list = []
+
+    for page in airtable.get_iter(view=view_name):
+        for record in page:
+            new_values = record["fields"]
+
+            if valid_entry(new_values, needed_fields):
```

Comment:
I think it's probably worth it to end up with a standard interface, but the previous code for teams was 
```
            if valid_entry(new_values):	
                if team_list.get(new_values.get("team")) is not None:	
                    team_list[new_values.get("team")].append(new_values.get("name"))	
                else:	
                    team_list[new_values.get("team")] = []	
```

so this will require a change to `about.vue` as well. I think we could probably just create a computed property based on the JSON value?

(we could also define callbacks or something, but I think it's probably better to have a standard form of the Airtable JSON and then process it per-page anyway)

Reply:
Good catch! I totally didn't notice that. Yeah I think having a standard JSON form is the way to go as well.